test-integration/helpers/task-plan-builder.coffee

```diff
@@ -285,5 +250,30 @@ class ReadingBuilder extends TestHelper
       when 'CANCEL' then @cancel()
       when 'DELETE' then @delete()
 
+  selectNumberOfExercises: (numExercises=4) ->
+    @_exerciseSelector.waitUntilLoaded()
+    @_exerciseSelector.selectNumberOfExercises(numExercises)
+
+  verifySelectedExercises: (numExercises) ->
+    @el.selectedExercises.findElements().then (els) ->
+      expect(numExercises).to.be.equal(els.length)
```

Comment:
I was not expecting to see `expect` outside of a test. Maybe this could be a method defined inside the test? Thoughts @openstax/tutor-fe ?